docs/octopus-rest-api/octopus-cli/clean-environment.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Cleaning environments
 description: Using the Octopus CLI to delete/remove machines with a particular status from environments on your Octopus instance.
-position: 10
+position: 20
```

Comment:
The reason we use gaps of 10 is so we can insert some in the middle without having to renumber everything :)
If you had just put the new one in as `5`, it would've had the same net effect :)

Reply:
ah  cool, so there might be some sort of styling around this `position` thing. Will revert most of the changes. Cheers :)